README.md

```diff
@@ -46,6 +46,18 @@ Arguments:
 - `size_display` tuple (int, int): width and height of the canvas (minimum 400)
 - `size_world` float: extend on the 3D scene in all directions
 
+Example: 
+
+```python
+from openalea.plantgl.all import *
```

Comment:
Sorry for being overly correct. I don't like the import *. Could you just import what you need? especially in a tutorial :)

Reply:
done